src/app/modules/group/pages/current-user/current-user.component.html

```diff
@@ -1 +1,81 @@
-<div>Yourself page works</div>
+<ng-container *ngIf="currentUser$ | async as currentUser">
+  <div class="alg-page-header">
+    <div class="alg-page-header-title">{{ currentUser.firstName }} {{ currentUser.lastName }}</div>
+  </div>
```

Comment:
Could you try to have the page title a bit larger than section title? A bit like [here](https://france-ioi.github.io/algorea-designs/01.Groups_06/Personal-data). Of course, we'll have to check that it is ok on the other pages as well, but that should make it.

Reply:
Fixed